index.js

```diff
@@ -22,10 +23,16 @@ module.exports = function (opts) {
 			from: file.path,
 			to: file.path
 		}).then(function (res) {
+			var map;
 			file.contents = new Buffer(res.css);
 
 			if (res.map && file.sourceMap) {
-				applySourceMap(file, res.map.toString());
+				map = res.map.toJSON();
+				map.file = file.relative;
+				map.sources = [].map.call(map.sources, function (source) {
+					return path.join(path.dirname(file.relative), source);
+				});
```

Comment:
They added it in this commit: https://github.com/postcss/gulp-postcss/commit/9feb6d4f99b705a4e9070abbd7dfa06a2d31b3b2#diff-168726dbe96b3ce427e7fedce31bb0bc to fix the relative sourcemaps.